_config.yml

```diff
@@ -234,8 +234,12 @@ social:
 
 lsi: false
 
+# Enable incremental builds by default
 incremental: true
 
+# Add GitHub Metadata
+github: [metadata]
```

Comment:
What's this?  Can you add a link in the comment?

Reply:
The CI was failing due to "No GitHub API authentication" error. 

Adding that line solved the [issue](https://github.com/github/pages-gem/issues/399).